README.md

```diff
@@ -2,55 +2,31 @@
 
 # Project Name
 
-> Description the project.
+> Set up a "Hello Microverse" project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS
+- VSC, github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://www.youtube.com/watch?v=UTga0ARWu2g&t=252s)
```

Comment:
- [ ] Kindly remove this section since you don't have a live demo of the project 👍

Reply:
done

Reply:
done